src/lib/images.ts

```diff
@@ -161,8 +161,8 @@ export const IMAGES = {
   LOGOUT_ICON_WHITE: 'logout-icon-white.svg',
   LOGOUT_ICON_BLACK: 'logout-icon-black.svg',
 
-  PAGE_NOT_FOUND: '404_b53b4ee6a2.svg',
-  SERVER_ERROR: '500_fe8e08a929.svg',
```

Comment:
if the domain is same for all strapi images then you can store this in variable and then you can use